### DISCUSSION TITLE
Is this still maintained?

### DISCUSSION THREAD

**Discussion by timelyart** [kinkofer/FightClub5eXML#193] [General]
Looking at the recent activity of this project, is this still maintained?

**Comment by leefujin** [comment]
As far as I know, unfortunately not

**Comment by zcdziura** [comment]
More than likely not. I've had a PR open for about a year that hasn't been merged.

**Comment by kinkofer** [comment]
I've been stalled on supporting new formatting changes where I had been getting my sources, but I hope to get back to this soon. In the meantime I've reviewed @zcdziura's PR and will make a few QoL updates after it's merged.

**Reply by zcdziura** [reply]
Thanks for the reply, @kinkofer. If there is any assistance that I can offer with the formatting changes you mentioned, I'm more than willing to help. I believe I know where you are getting your sources from and have also looked into the (ahem) byzantine format that's used there. I can offer whatever help necessary, including coding up some new features.

**Comment by timelyart** [comment]
Thanks for getting back to the community. 

Just to be clear, your intention is to keep maintaining the project. If so, is there anything we can do to help?

**Comment by kinkofer** [comment]
If anyone wants to help, one of [my repos](https://github.com/kinkofer/5eTtoFC5) is a fork off a project that converts json source data to an xml format used by the app Encounter+. This shares a similar xml format with Fight Club, but that app is for DMs so lacks a lot of the class features that Fight Club supports. The goal of that project is to pull the json sources and produce the xml files in the format needed for this project, or as close as possible with manual tweaking if needed.

I don't consider Python as part of my skillset (nor xml for that matter), so I learn as I go, and when I have time. There's also the added complexity of the collection format that I created to break out source books into their own files. Although that adds a little inconvenience, I think it's what's allowed this data to fly under the radar.

**Comment by abc** [comment]
This repo is unfortunately abandoned and development has moved over to ogre-project/FightClub5eXML for the time being, at least until @kinkofer returns and adds some contributors to this repo.
